packages/browser-wallet/src/popup/popupX/shared/utils/helpers.ts

```diff
@@ -1,4 +1,6 @@
-import { CcdAmount, Ratio } from '@concordium/web-sdk';
+import cbor from 'cbor';
```

Comment:
also have a look at https://github.com/hildjj/cbor2 as a replacement here. It states on the docs of https://www.npmjs.com/package/cbor that it's no longer being actively maintained. It's also not meant for use in a web context, but we probably polyfilled our way out of that originally